tests/Integration/Federation/FederationSchemaTest.php

```diff
@@ -8,6 +8,10 @@
 
 final class FederationSchemaTest extends TestCase
 {
+    protected string $federationV2SchemaExtension = /** @lang GraphQL */ <<<'GRAPHQL'
```

Comment:
`const`

Reply:
I've changed to `const` and concatenated with the rest of the schema definition